hack/build.sh

```diff
@@ -0,0 +1,89 @@
+# To push to your own registry, override the REGISTRY and NAMESPACE env vars,
+# i.e:
+#   $ REGISTRY=quay.io NAMESPACE=yourusername ./hack/build.sh
+#
+# REQUIREMENTS:
+#  * a valid login session to a container registry.
+#  * `docker`
+#  * `jq`
+#  * `yq`
+#  * `opm`
+#  * `skopeo`
+
+set -e
+
+export OPERATOR_NAME='observability-operator'
+export REGISTRY=${REGISTRY:-'quay.io'}
+export NAMESPACE=${NAMESPACE:-'rhobs'}
+export TAG=${TAG:-'0.0.23'}
+export CSV_PATH=${CSV_PATH:-'bundle/manifests/observability-operator.clusterserviceversion.yaml'}
+export ANNOTATIONS_PATH=${ANNOTATIONS_PATH:-'bundle/metadata/annotations.yaml'}
+
+function cleanup {
```

Comment:
nit: could you please use the same convention used in other scripts in the project? 

```suggestion
cleanup() {
```